### ISSUE TITLE
Suggestion: more minimal look

### ISSUE THREAD

**Issue by sepsol** [McJty/TheOneProbe#341]
- decrease the spaces between lines in TOP
- make the corners of TOP box rounded
- add the theme seen below from HWYLA
- overall, make it more minimal and add smaller sized HUD option for it to be more like HWYLA (please!)

![image](https://user-images.githubusercontent.com/47871887/68113935-23fc3d80-ff0a-11e9-9b28-cbfc809894d1.png)

**Comment by rckymtnrfc** [comment]
You can do a lot of that via the config file.  You can see I've edited mine to be in the lower corner, smaller and have no background or border.
![2019-11-16_11 25 57](https://user-images.githubusercontent.com/6869242/68997459-f6958500-0863-11ea-8829-8140967c51c6.png)

**Comment by sepsol** [comment]
True, but line spaces, making the corners of HUD element rounded, and having the "DARK" theme from HWYLA (can be turned on via the HWYLA's config file) is not available.

**Comment by McJty** [comment]
You can configure all the colors so you should be able to make a dark theme. Also with /topcfg you can configure a few looks including one that resembles the old WAILA

**Comment by sepsol** [comment]
Thanks for the info
